### PR TITLE
feat: issue_04

### PR DESCRIPTION
**Pull request by yoshikawa-akr**
# [issue No.4](https://github.com/kenmori/handsonFrontend/blob/master/work/markup/1.md#4)

**issue**
```
[{to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"}, {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}]
```
という配列を使って以下のようなHTML出力にしてください
```
<ul>
 <li><a href="/bookmark.html"><img src="1.png" alt="画像1">ブックマーク</a></li>
 <li><a href="/message.html"><img src="2.png" alt="画像2">メッセージ</a></li>
</ul>
```

## In terms of implementation, what you are worried about, what you want to focus on, and what you want to ask.
09e6c6c3474b8d48070835e761e7be13657cc2cf
I tried using "for of" for loop processing.

### CODE REVIEW

File: issue_04/script.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    const img = document.createElement('img');
+
+    a.textContent = item.text;
+    a.href = item.to;
+    img.src = item.img;
+    img.alt = item.alt;
+
+    fragment.appendChild(li).appendChild(a).insertAdjacentElement('afterbegin', img);
+}
+document.body.insertAdjacentElement('afterbegin', ul).appendChild(fragment);
```
Comment (seeeen044) [inline issue_04/script.js]: Question

Why did you omit HTML ul tag and add DOM to the body tag?

File: issue_04/script.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+const contentsArray = [{
```
Comment (hippopo-moo) [inline issue_04/script.js]: IMO
You can tell from the value that it is an array, so the variable name might as well be "contents".

Comment (yoshikawa-akr) [reply issue_04/script.js]: It's true that the value shows that it is an array, so we should review the array name. 
Thank you very much.
https://github.com/yoshikawa-akr/js-lessons/pull/4/commits/5febcd6735a7b1bad6987907b8ec43f34b721951

File: issue_04/script.js (hunk 1 above)
Comment (yoshikawa-akr) [reply issue_04/script.js]: I thought the task was to build the DOM in JavaScript, including the ul tag.

**Review by hippopo-moo — APPROVED** [review]

File: issue_04/script.js (hunk 1 above)
Comment (seeeen044) [reply issue_04/script.js]: Thank you for the answer.
As a code, I thought it would be easier to read if you installed ul like in issue 3.

**Review by seeeen044 — APPROVED** [review]

**Pull request merged**